src/chat.rs

```diff
@@ -4008,21 +4081,33 @@ pub(crate) async fn add_contact_to_chat_ex(
             );
             return Ok(false);
         }
-        if is_contact_in_chat(context, chat_id, contact_id).await? {
-            return Ok(false);
-        }
```

Comment:
I removed this block because we're in the `else` clause of `if is_contact_in_chat(context, chat_id, contact_id).await?`, so, this was dead code